scripts/zones/Outer_Horutoto_Ruins/mobs/Two_of_Swords.lua

```diff
@@ -0,0 +1,12 @@
+-----------------------------------
+-- Area: Outer Horutoto Ruins
+--  Mob: Two of Swords
+-----------------------------------
+local entity = {}
+
+entity.onMobDeath = function(mob, player, optParams)
+    xi.amk.helpers.cardianOrbDrop(mob, player, xi.ki.ORB_OF_SWORDS)
+    xi.regime.checkRegime(player, mob, 665, 3, xi.regime.type.GROUNDS)
```

Comment:
Not seeing any regime pages that use Two of Swords
https://www.bg-wiki.com/ffxi/Grounds_of_Valor#Outer_Horutoto_Ruins

Watch what you copy and paste. Make sure the thing you add actually belong in new mod scripts.